content/geoip/release-notes/2024.mdx

```diff
@@ -7,14 +7,41 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
-<ReleaseNote date="2024-02-13" title="No GeoNames Monthly Diff Report for February 2024">
+<ReleaseNote date="2024-03-13" title="API policies are now permanently enforced">
 
-Due to irregularities in the latest GeoNames data, we will be maintaining the
-location names and codes from January.
-[See the Monthly Diff Report for January 2024.](/geoip/release-notes/2024#geonames-monthly-diff-report-january-2024)
+To improve our server infrastructure and allow for better performance and
+efficiency, our API policies are now being permanently enforced as of March
+13, 2024.
 
-When the irregularities in the GeoNames data are resolved, we will update the
-data and post a new Diff Report.
+**What are the policies?**
+
+- MaxMind only accepts API and database download requests sent with the more
+  secure HTTPS protocol.
+- MaxMind only accepts API and database download requests that are sent to the
+  appropriate hostname as documented in the integration instructions on our
+  Developer Portal (see direct links below).
+
+**What do I need to do?** If you download GeoIP or GeoLite databases and you use
```

Comment:
need a double line break between `**What do I need to do?**` and `If you download` so that there's a paragraph break